### PR TITLE
adjust plugin order

### LINKED ISSUE #298: Categorizing plugins
### What would you like to be added?

Adjust the sequence of the plugin directory

### Why is this needed?

categorizing plugins

### PR DESCRIPTION
**Pull request by daizhenyu**
What type of PR is this?

Doc

Which issue(s) this PR fixes:

Fixes #298

### CODE REVIEW

File: docs/zh/document/plugin/README.md [hunk 1]
```diff
@@ -2,48 +2,58 @@
...
 
+### 基本服务注册发现和实时配置类：
+
 [动态配置插件](./dynamic-config.md)：基于Sermant配置中心能力实现动态配置，可在运行时将配置刷新到宿主应用，其优先级将高于环境变量配置。
 
+[SpringBoot注册插件](./springboot-registry.md)：纯SpringBoot应用提供服务注册发现能力，方便用户在不修改代码的前提下快速接入注册中心（目前只支持**Zookeeper**），同时提供超时重试的能力，实现服务调用的高可用。
+
+[注册迁移插件](./register-migration.md)：提供代码非侵入方式，可让原本注册于Eureka，Nacos，Zookeeper、Consul等主流注册中心的微服务，
+非侵入地注册到[ServiceComb](https://github.com/apache/servicecomb-service-center)，或[Nacos](https://nacos.io/)上,
+同时支持Dubbo与SpringCloud框架。
```
Comment (lilai23) [inline docs/zh/document/plugin/README.md]: Unnecessary line breaks

File: docs/en/document/plugin/README.md [hunk 2]
```diff
@@ -2,30 +2,45 @@
...
 
-[Registry Plugin](./service-registry.md) allows microservices that have been registered with popular registration centers, such as Eureka, Nacos, ZooKeeper, and Consul, to be registered with [ServiceComb](https://github.com/apache/servicecomb-service-center) or [Nacos](https://nacos.io/) in a non-intrusive manner. It also supports Dubbo and SpringCloud frameworks.
+### Application Observable categories:
+
+[Monitor Plugin](./monitor.md) can monitor the usage of hardware resources such as CPU, memory, disk IO and network IO of the server where the host application is located, as well as the usage of the host application Java virtual machine and the Druid database connection pool used by it.
+
+[[Service visibility Plugin](./visibility.md)]: Provides contract information and blood relationship collection and 
+display functions for Spring Cloud and Dubbo applications, allowing users to view the interface information provided 
+by all services and the calling relationship information between services through the Sermant Backend without modifying 
+the code.
```
Comment (lilai23) [inline docs/en/document/plugin/README.md]: Unnecessary line breaks

File: docs/zh/document/plugin/README.md (hunk 1 above)
Comment (daizhenyu) [reply docs/zh/document/plugin/README.md]: done

File: docs/en/document/plugin/README.md (hunk 2 above)
Comment (daizhenyu) [reply docs/en/document/plugin/README.md]: done

**Pull request merged**